yapapi/engine.py

```diff
@@ -102,8 +102,8 @@ def __init__(
         budget: Union[float, Decimal],
         strategy: Optional[MarketStrategy] = None,
         subnet_tag: Optional[str] = None,
-        driver: Optional[str] = None,
-        network: Optional[str] = None,
+        payment_driver: Optional[str] = None,
```

Comment:
those parameters should stay here as deprecated

Reply:
Again, why do we need deprecAtions on `Engine`? I think only on `Golem`.

Reply:
okay, yeah, they don't need to be there on `_Engine` ... still, the comment stays the same with regards to the Golem class